scripts/rules/update_tasks.sh

```diff
@@ -148,6 +148,7 @@ else
   rm "$SETTINGS_TMP_FILE"
 fi
 
+# TODO fix this
```

Comment:
And this?

Reply:
Removed TODO. Old API is fine for now. I suspect these scripts are useless now anyway. I'm going to go through and clean them out soon...